[0m01:00:33.360676 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B326BC9040>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B32655F6E0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B32655FF50>]}


============================== 01:00:33.388827 | df4c01ab-4f20-422d-80ff-dbafd731435b ==============================
[0m01:00:33.388827 [info ] [MainThread]: Running with dbt=1.8.7
[0m01:00:33.393065 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\Hp\\.dbt', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'warn_error': 'None', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'invocation_command': 'dbt init', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m01:00:33.542730 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': WindowsPath('C:/Users/Hp/.dbt')}
[0m01:00:33.547766 [info ] [MainThread]: Creating dbt configuration folder at 
[0m01:02:23.778480 [debug] [MainThread]: Starter project path: C:\Users\Hp\PycharmProjects\HelloWorld\venv\Lib\site-packages\dbt\include\starter_project
[0m01:02:23.854361 [info ] [MainThread]: 
Your new dbt project "ETH_MedData_Warehouse" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m01:02:23.860681 [info ] [MainThread]: Setting up your profile.
[0m01:09:08.325374 [info ] [MainThread]: Profile ETH_MedData_Warehouse written to C:\Users\Hp\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m01:09:08.330989 [debug] [MainThread]: Command `dbt init` succeeded at 01:09:08.328917 after 515.76 seconds
[0m01:09:08.333769 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B326AEFC50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B327007710>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B326D83560>]}
[0m01:09:08.335389 [debug] [MainThread]: Flushing usage events
[0m11:36:06.249033 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A183544500>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A185DA20C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A185E68CE0>]}


============================== 11:36:06.274674 | da2d2a5d-cffb-4172-8c30-12cbf7b06e8f ==============================
[0m11:36:06.274674 [info ] [MainThread]: Running with dbt=1.8.7
[0m11:36:06.278219 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\Hp\\.dbt', 'debug': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run', 'log_format': 'default', 'introspect': 'True', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m11:36:06.280171 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\Hp\Desktop\Kifiya\Week7\ETH-MedData-Warehouse\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m11:36:06.289432 [debug] [MainThread]: Command `dbt run` failed at 11:36:06.288421 after 0.44 seconds
[0m11:36:06.291425 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A183544500>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A185028680>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A185F03E60>]}
[0m11:36:06.293379 [debug] [MainThread]: Flushing usage events
[0m15:30:12.686104 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001CA8F686840>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001CA8F686510>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001CA8F686A20>]}


============================== 15:30:12.711958 | ba7708c9-6aa9-4186-9871-9c664a1b8313 ==============================
[0m15:30:12.711958 [info ] [MainThread]: Running with dbt=1.8.7
[0m15:30:12.711958 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'profiles_dir': 'C:\\Users\\Hp\\.dbt', 'log_path': 'logs', 'debug': 'False', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'invocation_command': 'dbt run', 'send_anonymous_usage_stats': 'True'}
[0m15:30:12.717500 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\Hp\Desktop\Kifiya\Week7\ETH-MedData-Warehouse\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m15:30:12.729168 [debug] [MainThread]: Command `dbt run` failed at 15:30:12.726884 after 0.38 seconds
[0m15:30:12.731029 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001CA8F42C980>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001CA8F6DA990>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001CA8ED8FFB0>]}
[0m15:30:12.733560 [debug] [MainThread]: Flushing usage events
